### PR TITLE
Accordion complete

### PR DESCRIPTION
**Pull request by sanketXynes**
Accordion

### CODE REVIEW

File: src/components/Disclosuers/Accordion/accordion.styles.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  transform-origin: center;
+  cursor: pointer;
+}
+
+.rotated {
+  transform: rotate(90deg);
+}
+
+.lmAccordionBody {
+  margin-top: 1vh;
```
Comment (archan96) [inline src/components/Disclosuers/Accordion/accordion.styles.css]: Can we have this in Styled component with margin-top: ${spacing.padding.small} like that?

**Review by archan96 — COMMENTED** [review]
Please do the changes and push.

**Review by archan96 — APPROVED** [review]
Looks alright

**Review by xynes-admin — APPROVED** [review]
All seems fine.

**Pull request merged**